mount/Makefile

```diff
@@ -1,3 +1,2 @@
 titus-mount: mount.c
-	gcc -D_GNU_SOURCE=1 -std=gnu11 -Wall -Werror -static -g -o titus-mount mount.c
-
+	musl-gcc -std=gnu11 -Wall -static -g -o titus-mount mount.c
```

Comment:
Does musl have the same behaviour of glibc as to reading /etc/resolv.conf on the first query?

Reply:
I don't know, but this is the only dns query this code does, should be fine right?
There are some documented differences, but nothing about resolv.conf: https://wiki.musl-libc.org/functional-differences-from-glibc.html#Name-Resolver/DNS

You did mention there was something I could add that I could test, what was that again?

Reply:
musl doesn't implement the low level libresolv API. Reading the musl code, it looks like the behaviour is the same, where they load resolv.conf on the first query